tests/pl/climate_HassClimateSetTemperature.yaml

```diff
@@ -15,6 +16,7 @@ tests:
       slots:
         temperature: 30
         temperature_unit: "celsius"
+    response: Ustawiono temperaturę na 30° stopni
```

Comment:
Czy tu ten symbol "°" powinien wciąż być?

Reply:
yes, note that the tests cover the temperature change with and without this symbol and this is intentional